{{cookiecutter.project_slug}}/Pipfile

```diff
@@ -11,7 +11,7 @@ python_version = "3.8"
 [packages]
 cryptography = "~=37.0.2"
 django = "~=2.2.26"
-"boto3" = "~=1.24.13"
+"boto3" = "~=1.24.22"
```

Comment:
This package is not related to this PR, perhaps we should leave it untouched?

Reply:
Yes, it would be better handle dependencies update in other PR

Reply:
This was because of a rebase, a dependabot commit ended up the branch - now the rebase should be proper

Reply:
Okay, great. Thanks!